### PR TITLE
startRecording config parameter and auto recording start changes added

### PR DESCRIPTION
**Pull request by harishnv**

### CODE REVIEW

File: JitsiConference.js [hunk 1]
```diff
@@ -1825,6 +1827,43 @@ JitsiConference.prototype.muteParticipant = function(id, mediaType) {
     this.room.muteParticipant(participant.getJid(), true, muteMediaType);
 };
 
+
+JitsiConference.prototype.onStartRecording = function(data) {
+
```
Comment (manquer) [inline JitsiConference.js:1832]: try catch ?

File: JitsiConference.js [hunk 2]
```diff
@@ -2035,6 +2074,7 @@ JitsiConference.prototype.onMemberKicked = function(
  */
 JitsiConference.prototype.onLocalRoleChanged = function(role) {
     // Emit role changed for local  JID
+    this.onStartRecording({ test: true });
```
Comment (manquer) [inline JitsiConference.js]: test true ?

**Review by manquer — CHANGES_REQUESTED** [review]
There is no dep change why lock file changed?

File: JitsiConference.js (hunk 1 above)
Comment (harishnv) [reply JitsiConference.js:1832]: implemented

File: JitsiConference.js (hunk 2 above)
Comment (harishnv) [reply JitsiConference.js]: updated

**Review by manquer — APPROVED** [review]

**Pull request merged**